src/LaunchDarkly.Client/FeatureFlag.cs

```diff
@@ -0,0 +1,220 @@
+﻿using System.Collections.Generic;
+using LaunchDarkly.Client.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace LaunchDarkly.Client
+{
+    public class FeatureFlag
```

Comment:
This is public because we want to access it in tests, but I just found this: https://msdn.microsoft.com/en-us/library/system.runtime.compilerservices.internalsvisibletoattribute.aspx which may allow us to make it internal...